bvm/ballerina-core/src/main/java/org/ballerinalang/bre/old/BaseWorkerResponseContext.java

```diff
@@ -164,4 +165,19 @@ protected void doFailCallbackNotify(BError error) {
             }
         }
     }
+
+    void signalTransactionError(WorkerSignal signal) {
+        TransactionLocalContext transactionLocalContext = signal.getSourceContext().getLocalTransactionInfo();
+        if (transactionLocalContext == null) {
+            return;
+        }
+        transactionLocalContext.notifyLocalParticipantFailure();
+    }
+
+    void signalTransactionParticipantSuccess(WorkerSignal signal) {
+        TransactionLocalContext transactionLocalContext = signal.getSourceContext().getLocalTransactionInfo();
+        if (transactionLocalContext == null) {
+            return;
```

Comment:
No point of adding this return here.

Reply:
BaseWorkerResponseContext.java will be deleted as it's no longer used in new BVM class